src/components/common/BookCard/BookCard.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Button from '../Button/Button';
+import CardImage from '../CardImage/CardImage';
+import Typography from '../Typography/Typography';
+import './BookCard.scss';
+
+const BookCard = ({ children, book }) => {
+  return (
+    <Link to={`/book/${book._id}`}>
+      <div className="BookCard__root">
+        <CardImage url={book.image} width="100%" height="70%" />
+        <div className="BookCard__content">
+          {children || (
+            <>
+              <div className="text-black">
+                <Typography variant="h6" className="BookCard__title">
+                  {book.title}
+                </Typography>
+                <div className="d-flex">
+                  {book.categories.map((item, idx) => (
+                    <p key={idx} className="text-primary">
+                      {item}
+                    </p>
```

Comment:
Suggestion: Try not to use index for key as when array is sorted there could be error in diffing list items and thus some items may or may not update.

reference: 
Key section in https://reactjs.org/docs/reconciliation.html

Reply:
Not using **index as key** is recommended when the order of items change dynamically. 
Example: A Todo where items change/ update their order based on user interaction. 
Here, the items order donot change once they have been rendered. So index as key can be used here.